unit_tests/test_request_object.py

```diff
@@ -1,14 +1,14 @@
-from robyn.robyn import Headers, QueryParams, Request, Url
+from robyn.robyn import Headers, RustQueryParams, Request, RustUrl
 
 
 def test_request_object():
-    url = Url(
+    url = RustUrl(
         scheme="https",
         host="localhost",
         path="/user",
     )
     request = Request(
-        query_params=QueryParams(),
+        query_params=RustQueryParams(),
```

Comment:
Should be just QueryParams

Reply:
A follow up note to custom types - if a custom type is not instantiable and a drop in replacement, then there is something wrong with our approach